src/main/java/edu/kis/powp/jobs2d/drivers/ScaledLineDriver.java

```diff
@@ -0,0 +1,61 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.legacy.drawer.panel.DrawPanelController;
+import edu.kis.legacy.drawer.shape.ILine;
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.transformations.DriverLevelTransformation;
+
+/**
+ * Line adapter - Job2dDriver with DrawPanelController object.
+ */
+public class ScaledLineDriver implements Job2dDriver {
+	private ILine line;
+	private int startX = 0, startY = 0;
+	private String name;
+	private DriverLevelTransformation transformation;
+
+	private DrawPanelController drawController;
```

Comment:
You should not reimplement a drawing driver again. Use the decorator pattern to re-use already existing drivers!